client/lib/signup/step-actions/index.js

```diff
@@ -281,8 +285,8 @@ export function createSiteWithCart( callback, dependencies, stepData, reduxStore
 		saveToLocalStorageAndProceed( state, domainItem, themeItem, newSiteParams, callback );
 		return;
 	}
-
 	const locale = getLocaleSlug();
+	const isNewUser = getCurrentUserSiteCount( state ) <= 1;
```

Comment:
Should we also check if the user is < 7 days old?

Reply:
You're right! I'll update the PR with a <7 day check 👍

Reply:
Updated, but I'm not 100% sure about the conditional - since we're here firing it inside the response handler, so I think we should check the site count, and then if they registered within the last 7 days. 

Because in the current code, in `main.jsx` we also check for site slug and a few more things, but we shouldn't need that here, since we can be pretty confident a site is being created.

I'll do some more testing to.

Reply:
> Because in the current code, in main.jsx we also check for site slug and a few more things, but we shouldn't need that here, since we can be pretty confident a site is being created.

Right, since the site is already created, we don't need to check for siteSlug, so checking for site count and if it's a new 7D user should be good.